pallets/runtime/develop/src/runtime.rs

```diff
@@ -671,7 +671,7 @@ impl asset::Trait for Runtime {
 }
 
 parameter_types! {
-    pub const MaxConditionComplexity: u32 = 50;
+    pub const MaxConditionComplexity: u32 = 150;
```

Comment:
Let's lower this back to 50, create examples of what can fit in 50, and ask for product feedback.

Reply:
I increase this to see the effect of the worst condition. I think this is something we can get from CM benchmarks.

Reply:
Can you please create a JIRA ticket for this?